jobs/ClusterHQ_SeniorEngineer_perm_techfolk_2.html

```diff
@@ -1,5 +1,5 @@
 ---
-title: Senior & Lead Software Engineer
+title: Lead Software Engineer: storage | containers | Docker
```

Comment:
I think this line is the cause of the failing build. YAML snafu: the colon makes this value into a dict! Try wrapping the whole value in double quotes.